crates/hc_sandbox/src/cmds.rs

```diff
@@ -44,6 +44,10 @@ pub struct Create {
     #[arg(long, action = ArgAction::SetFalse)]
     pub no_dpki: bool,
 
+    /// Set the network seed for the DPKI service.
+    #[arg(long)]
+    pub dpki_network_seed: Option<String>,
```

Comment:
Minor suggestion, tries to prevent redundant setting of `--dpki-network-seed` when `--no-dpki` is already set

```suggestion
    #[arg(long, requires_if("!no_dpki", "no_dpki"))]
    pub dpki_network_seed: Option<String>,
```

Reply:
@c12i I tried adding this but it seems to have no effect. I'm not sure what the proper flag is, could you add a commit which creates the proper requirement restriction?

Reply:
My mistake, the correct rule would be `conflicts_with` rather than `requires_if`. Pushed the commit